packages/vitest/src/runtime/runVmTests.ts

```diff
@@ -21,7 +22,8 @@ import { getWorkerState } from './utils'
 
 export async function run(
   method: 'run' | 'collect',
-  files: string[],
+  // TODO consider type
```

Comment:
todo

Reply:
Just wasn't sure if this was a public interface or not